Darwin/Services/StitchRemoteMongoDBService/StitchRemoteMongoDBServiceTests/FoundationInstrumentedHTTPTransport.swift

```diff
@@ -0,0 +1,129 @@
+import Foundation
+import MongoSwift
+@testable import StitchCoreSDK
+
+/**
+ * A basic implementation of the `Transport` protocol using the `URLSession.dataTask` method in the Foundation library.
+ */
+public class FoundationInstrumentedHTTPTransport: FoundationHTTPTransport {
+    private var bytesUploadedForNormalRequests: Int64 = 0
+    private var bytesDownloadedForNormalRequests: Int64 = 0
```

Comment:
[nit] Can make both of these unsigned– use your best judgment for if that should be 32 or 64 bits.

Reply:
Interestingly enough, Foundation gives the byte counts as a signed int:
https://github.com/phracker/MacOSX-SDKs/blob/master/MacOSX10.9.sdk/System/Library/Frameworks/Foundation.framework/Versions/C/Headers/NSURLSession.h#L243

I'm going to leave this as is, because it feels unsafe to cast a signed long to an unsigned long from a library that explicitly gives a signed long